### ISSUE TITLE
Warnings after after 2.2.1 update

### ISSUE THREAD

**Issue by andrejcesen** [nkt/atom-autocomplete-modules#118]
### Description

After updating package to 2.2.1 each time module autocomplete pops up (it works), I also get an Atom warning - see the attached image.

<img width="521" alt="screenshot of atom 05-10-2018 10-14-52" src="https://user-images.githubusercontent.com/1460733/46524071-f7e63700-c887-11e8-9ec5-f7f147348a68.png">

I'm working in React Native, here's my `.babel.rc`:
```
["module-resolver", {
  "root": ["."],
  "extensions": [".ios.js", ".android.js", ".js", ".json"]
}]
```

### Steps to Reproduce

1. On version 2.2.0 I am not getting any warnings on this line:
`import { colors }  from 'src/globalStyle';`
2. On version 2.2.1 I am not getting the above warning message 

#### Folder structure and import statement

My folder structure looks like this:
```
app/
├── src/
│   ├── GlobalStyle.js
```

```javascript
// app/src/current.js

import { colors }  from 'src/G| ';
```

**Expected behavior:** No warnings

**Actual behavior:** Above warning

**Reproduces how often:** 100%

#### Console errors

```
(2) [{…}, "Error: ENOENT: no such file or directory, scandir …/node_modules/src/'"]0: {scope: "module-local", prefix: "./src/G", additional: Array(1)}1: "Error: ENOENT: no such file or directory, scandir '.../node_modules/src/'"length: 2__proto__: Array(0)

```

### Versions

```
Atom    : 1.31.1
Electron: 2.0.7
Chrome  : 61.0.3163.100
Node    : 8.9.3

apm  2.1.1
npm  6.2.0
node 8.9.3 x64
atom 1.31.1
python 2.7.15
git 2.16.1
```

**Comment by jonyeezs** [comment]
Hi,

tl;dr: you may have your root wrong. set it to `"root": ["./app"]`

I noticed that you have your root as `'.'`, but your folder starts at `./app`.

You're trying to import from `src/...` which isn't a dependency but a babel alias/root.

In this case no alias provided so it must be a root. Unfortunately your root doesn't start within `./app` but at the same level, which the folder `src` doesn't exist.

Hence, the lookup failure.

**Comment by jonyeezs** [comment]
i will as well remove the notifications. They don't serve any good purpose.
